wordpress-mobile/WordPress-Android
# [Blogging prompts] Add new analytics event for create sheet

Fixes #17689 

To test:
1 - Launch JP and log in;
2 - Select a site that is a blog;
3 - Select "My Site" and `HOME`;

4 - Tap the FAB 
<img width="92" alt="image" src="https://user-images.githubusercontent.com/14964993/209448390-7f6a2346-7bf8-4764-9fd5-faedc1b4187f.png">

6 - The create sheet should be displayed. Logcat should show that `blogging_prompts_create_sheet_card_viewed` was tracked:
`I/WordPress-STATS: 🔵 Tracked: blogging_prompts_create_sheet_card_viewed`

## Regression Notes
1. Potential unintended areas of impact
None

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing and unit tests

3. What automated tests I added (or what prevented me from doing so)
Updated `MySiteViewModelTest.kt`, `BloggingPromptsCardAnalyticsTrackerTest.kt` and `WPMainActivityViewModelTest.kt`.

PR submission checklist:
